yoshi/tests/test_yoshi.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from yoshi.yoshi import run_one_yoshi
```

Comment:
And this should probably use a relative import?

Reply:
There is debate on that.  The most recent astropy consensus (and PEP8) is to use absolute.